### PR TITLE
USBAT to see a flat's page with details

### PR DESCRIPTION
**Pull request by richiebeans**

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/flats.js.erb [hunk 1]
```diff
@@ -13,7 +14,7 @@ app.controller('ListRoomCtrl', ['$scope', '$http', '$auth', '$location', functio
     Name: 'House'
     }, {
     Id: 3,
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/flats.js.erb:16]: you guys never fixed the indentation here?

File: app/assets/javascripts/angular/controllers/flats.js.erb [hunk 2]
```diff
@@ -78,9 +79,13 @@ app.controller('ListRoomCtrl', ['$scope', '$http', '$auth', '$location', functio
     var data = {
       flat: $scope.flat
     };
-
     $http.post(url + '/flats', data).success(function(response){
       console.log(response);
     });
   };
 }]);
+
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/flats.js.erb:87]: extra spaces at the end of file... yoooo

**Comment by raphaeldieu** [comment]
All of this had been fixed on the next PRs promise !!!!

File: app/assets/javascripts/angular/controllers/singleflat.js.erb [hunk 3]
```diff
@@ -0,0 +1,49 @@
+app.controller('SingleFlatCtrl', ['$scope', '$http', '$auth', '$location', '$routeParams', function($scope, $http, $auth, $location, $routeParams){
+
+
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
+
+  $http.get(url + '/flats/' + $routeParams.id).success(function(response){
+    $scope.flat = response;
+    console.log(response)
+
+    if ($scope.flat.property_can_smoke == true){
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/singleflat.js.erb]: very useful: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

File: app/assets/stylesheets/singleflat.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+header.carousel .fill {
+    width: 100%;
+    height: 100%;
+    background-position: center;
+    background-size: cover;
+}
+
+    header.carousel .carousel {
```
Comment (f3r) [inline app/assets/stylesheets/singleflat.css]: indentation....

File: app/helpers/singleflat_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module SingleflatHelper
```
Comment (f3r) [inline app/helpers/singleflat_helper.rb]: delete if empty pls

File: spec/controllers/singleflat_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (f3r) [inline spec/controllers/singleflat_controller_spec.rb]: delete pls

File: spec/helpers/singleflat_helper_spec.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (f3r) [inline spec/helpers/singleflat_helper_spec.rb]: delete pls

**Comment by f3r** [comment]
fix comments and can merge, go go go!! :)

**Pull request merged**